app/validation/validator.py

```diff
@@ -15,7 +15,7 @@
 MAX_DECIMAL_PLACES = 6
 
 
-class Validator:    # pylint: disable=too-many-public-methods
+class Validator:    # pylint: disable=R0904, C0302
```

Comment:
This still irks me, AFAICS pylint should allow you to use multiple named rules.

Reply:
Yes me too, as I said - I've tried but couldn't get it to work when the rules were named.

Reply:
Huh, turned out the rule name I was using was incorrect as it was named differently in the error. I've updated this now.

Reply:
Nice.